### ISSUE TITLE
Losing certain columns

### ISSUE THREAD

**Issue by Risiamu** [iRon7/ConvertFrom-SourceTable#2]
Following is the output of `transmission-remote --auth xxx:Xxxx -t all -l| Select-Object -SkipLast 1`
```
ID     Done       Have  ETA           Up    Down  Ratio  Status       Name
   1   100%   17.34 MB  Unknown      0.0     0.0    0.0  Idle         BitComet Stable (build 1.69.7.8)
   2    31%   14.54 GB  50 min    1533.0  9600.0    0.0  Up & Down    [BeanSub&VCB-Studio] Vinland Saga [Ma10p_1080p]
   3   100%   75.47 MB  Unknown      0.0     0.0    0.0  Idle         [Airota][Kakushigoto][01-12][ASS][fonts][CHS].7z
   4     7%   353.4 MB  1 hrs        0.0  2285.0    0.0  Downloading  [Sakurato.sub][Kakushigoto][01-12 END][CHT][1080P]

```
after `transmission-remote --auth xxxxx:Xxxxx -t all -l| Select-Object -SkipLast 1 |ConvertFrom-SourceTable  -Literal`
the output is:
```
ID     :
Done   : 100%
Have   : 17.34 MB
ETA    : Unknown
Up     : 0.0
Down   : 0.0
Ratio  : 0.0
Status : Idle
Name   : BitComet Stable (build 1.69.7.8) 

ID     :
Done   : 28%
Have   : 13.03 GB
ETA    : 1 hrs
Up     : 1492.0
Down   : 9564.0
Ratio  : 0.0
Status : Up & Down
Name   : [BeanSub&VCB-Studio] Vinland Saga [Ma10p_1080p]

ID     :
Done   : 100%
Have   : 75.47 MB
ETA    : Unknown
Up     : 0.0
Down   : 0.0
Ratio  : 0.0
Status : Idle
Name   : [Airota][Kakushigoto][01-12][ASS][fonts][CHS].7z

ID     :
Done   : 6%
Have   : 289.5 MB
ETA    : 2 hrs
Up     : 0.0
Down   : 356.0
Ratio  : 0.0
Status : Downloading
Name   : [Sakurato.sub][Kakushigoto][01-12 END][CHT][1080P]

```

The data in ID column is missing.

**Comment by iRon7** [comment]
Hello GrandArth,

Thank you for notifying me on the bug.
I have resolved the issue in the new version 0.3.12 (just uploaded)

**Comment by Risiamu** [comment]
Thx.
This function really gets the job done.
